www/js/shortcuts/shortcuts.js

```diff
@@ -0,0 +1,18 @@
+/* global Mousetrap */
+const shortcutsMap = require('./shortcuts-map.json');
+
+const shortcutFactory = (keys, actionName) => {
+  Mousetrap.bindGlobal(keys, () => {
+    global.platform.ipc.send('shortcuts', actionName);
+  });
+};
+
+const applyShortcuts = () => {
```

Comment:
lets make sure the event handlers don't get duplicated if multiple external monitors are attached.